metadata-integration/java/datahub-client/src/main/java/datahub/client/rest/RestEmitter.java

```diff
@@ -154,7 +155,12 @@ public static RestEmitter createWithDefaults() {
   @Override
   public Future<MetadataWriteResponse> emit(MetadataChangeProposalWrapper mcpw,
       Callback callback) throws IOException {
-    return emit(this.eventFormatter.convert(mcpw), callback);
+    try {
+      return emit(StringEscapeUtils.convert(mcpw), callback);
+    } catch (URISyntaxException e) {
```

Comment:
Do we need to catch any IO exception?

Reply:
Let the caller handle ioexception. We cannt do anything about exception at this point.